src/iblt.h

```diff
@@ -118,8 +126,17 @@ class CIblt
     inline void SerializationOp(Stream &s, Operation ser_action)
     {
         READWRITE(COMPACTSIZE(version));
-        if (ser_action.ForRead() && version != 0)
-            throw std::ios_base::failure("Only IBLT version zero is currently known.");
+
+        if (version > 0)
+        {
+            READWRITE(mapHashIdxSeeds);
+            READWRITE(salt);
+            if (salt > VALS_32 / n_hash)
```

Comment:
move this test lower, since it is using n_hash which has not been read yet.

Reply:
Removed entirely in 4eee6a1.